docs/am43xx.md

```diff
@@ -0,0 +1,57 @@
+# OP-TEE on Texas Instruments AM43xx
```

Comment:
We also use a different first stage u-boot name:
u-boot-spl_HS_ISSW
vs
u-boot-spl_HS_MLO

I'll see if I can find a way to work around this in the readme